CMakeModules/Dependencies.cmake

```diff
@@ -1,8 +1,18 @@
-if(NOT WIN32)
+if(NOT BUILD_PLATFORM STREQUAL "win32")
   include_directories("/usr/local/include")
   link_directories("/usr/local/lib")
 endif()
 
+set(ENABLE_DIRECTDRAW OFF CACHE BOOL "Enable DirectDraw on Windows x86")
```

Comment:
We can do that. Previously it was done that way as there was an assumption that option should be defined in targets where it is used. But then we ended up with options which are used for multiple separate targets. Don't really want to clutter main `CMakeLists.txt` though... IMHO it should be as minimal as possible. Maybe better to move them to separate file like `CMakeModules/Options.cmake` and just include it in the beginning of main `CMakeLists.txt`?

Reply:
The purpose here is to have the options all in the same place so that users could see them just by reading the cmake file. I prefer to have them in the main CMakeLists.txt, but having a separate file just for options is also OK.

Reply:
Or they can just execute `cmake -L` or even `cmake -LA`...

Reply:
True, but `cmake -L` output is usually pretty cluttered.